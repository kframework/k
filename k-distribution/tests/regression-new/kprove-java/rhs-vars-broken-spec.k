requires "demo.k"

module VERIFICATION
    imports DEMO
endmodule

module RHS-VARS-BROKEN-SPEC
    imports VERIFICATION

    rule <k> .K => _ </k>

    rule <k> .K => A </k>
endmodule
