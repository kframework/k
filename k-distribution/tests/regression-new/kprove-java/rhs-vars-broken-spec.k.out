[Error] Compiler: Found variable _ on right hand side of rule, not bound on
left hand side. Did you mean "?_"?
	Source(rhs-vars-broken-spec.k)
	Location(10,20,10,21)
[Error] Compiler: Found variable A on right hand side of rule, not bound on
left hand side. Did you mean "?A"?
	Source(rhs-vars-broken-spec.k)
	Location(12,20,12,21)
[Warning] Compiler: Variable 'A' defined but not used. Prefix variable name
with underscore if this is intentional.
	Source(rhs-vars-broken-spec.k)
	Location(12,20,12,21)
[Error] Compiler: Had 2 structural errors.
